bee-protocol/src/worker/requester/milestone.rs

```diff
@@ -116,7 +116,7 @@ impl<'a> MilestoneRequesterWorker<'a> {
         }
 
         if retry_counts > 0 {
-            debug!("Retried {} milestones.", retry_counts);
+            info!("Retried {} milestones.", retry_counts);
```

Comment:
I know it gets lost a bit in debug (because we need to clean debug a bit), but I don't think this should end up being info though :(